src/App.tsx

```diff
@@ -63,6 +84,37 @@ const checkoutTheme = createTheme({
         },
       },
     },
+    MuiOutlinedInput: {
+      styleOverrides: {
+        root: {
+          borderRadius: "8px",
+        },
+      },
+    },
+    MuiLoadingButton: {
+      styleOverrides: {
+        loadingIndicator: {
+          color: "white !important",
```

Comment:
This !important can be removed, also the loadingIndicator should have a size of 24px